Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM node:18-alpine
+# stage 1 
+FROM node:18-alpine as builder
```

Comment:
I think this should be a debian.

Reply:
i think alpine is okay as its an independent distribution and a preference since its small close to only 5MB

Reply:
I think using a different base OS here vs what will be used afterwards is looking for problems. Use the same one for boths.

Reply:
I feel Apine is preferAble here... @HassanBahati 👍